docs/user-guide/spark-user-guide.md

```diff
@@ -290,9 +306,33 @@ entity "**vocabulary**" {
 Because we need the triplet of `concept_id`, `vocabulary_id`, and `vocabulary_version` to query the terminology provider,
 we have to join across both the `concept` and `vocabulary` tables.
 
-And we would use the `with` field within a `MultiManyToMany` join to represent this relationship:
+This would result in the following context definition:
 
-[MultiManyToMany example ContextDefinition](context-definition/example-multi-many-to-many.json ':include :type=json')
+```json
+{
+  "contextDefinitions": [
+    {
+      "name": "person",
+      "primaryDataType": "person",
+      "primaryKeyColumn": "person_id",
+      "relationships": [
+        {
+          "name": "concept",
+          "joinClause": "inner join observation on person.person_id = observation.person_id inner join concept on observation.observation_concept_id = concept.concept_id"
+        },
+        {
+          "name": "vocabulary",
+          "joinClause": "inner join observation on person.person_id = observation.person_id inner join concept on observation.observation_concept_id = concept.concept_id inner join vocabulary on concept.vocabulary_id = vocabulary.vocabulary_id"
+        },
+        {
+          "name": "observation",
+          "joinClause": "inner join observation on person.person_id = observation.person_id"
+        }
+      ]
+    }
+  ]
+}
+```
```

Comment:
Not necessarily a concern, but wanted to confirm an observation I had. It looks like each of these joinClause statements are meant to be independent of each other and all based on/relative to primaryDataType? Versus incremental and based on prior joinClause statements? Assume the code and produces the final SQL statement to use for a context is able to filter out redundant logic across multiple joinClause expressions?

Reply:
Your observation is correct. The joins work in the same manner that they did before these changes. Each join is independent.